components/infra-proxy-service/server/orgs.go

```diff
@@ -240,7 +241,11 @@ func (s *Server) GetInfraServerOrgs(ctx context.Context, req *request.GetInfraSe
 	// Save organisations in backend DB
 	orgs := []storage.Org{}
 	for key := range orgsList {
-		org, err := s.service.Storage.StoreOrg(ctx, key, key, "", "", req.ServerId, nil)
+		projects, err := createProjectFromOrgIdAndServerID(s, ctx, req.ServerId, key)
```

Comment:
Will it create the policies too for the project?

Reply:
Yes, we are passing skip policies as false, therefore its creating project with all the policies.